viewing_party/party.py

```diff
@@ -1,23 +1,252 @@
+from statistics import multimode,mode
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    """Create function to store movie
+    Create empty dictionary to store the movie
+    Check for edge case of one item being none
+    """
+
+    movie = { "title": "", "genre" : "", "rating" : 0}
+    if title is None or genre is None or rating is None:
+        return None
+    else:
+        movie["title"]= title

```

Comment:
[nit] add spaces before `=`

```suggestion
        movie["title"] = title

```